internal/dao/helm_chart.go

```diff
@@ -145,8 +145,21 @@ func (h *HelmChart) Uninstall(path string, keepHist bool) error {
 
 // ensureHelmConfig return a new configuration.
 func ensureHelmConfig(flags *genericclioptions.ConfigFlags, ns string) (*action.Configuration, error) {
+	settings := &genericclioptions.ConfigFlags{
```

Comment:
I don't think we need to clone all the flags. Setting the namespace should be sufficient since the namespace override is not considered.